### PR TITLE
express-get-json

### PR DESCRIPTION
**Pull request by eniolajohnson**
image

### CODE REVIEW

File: express-get-json/index.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+}
+
+const jsonArr = [];
```
Comment (Zhiwutian) [inline express-get-json/index.js:25]: This and the loop below must be in the .get method call code block.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed!

**Review by Zhiwutian — APPROVED** [review]
👍

File: express-get-json/index.js (hunk 1 above)
Comment (eniolajohnson) [reply express-get-json/index.js]: image

Done! May I please know why? I couldn't tell the difference between declaring and running the for...in loop outside and within the app.METHOD.

**Pull request merged**